tools/jit/gen_jit_dispatch.py

```diff
@@ -33,16 +33,18 @@
 """)
 
 CALL_NAMESPACE = CodeTemplate("at::${name}(${args})")
-CALL_METHOD = CodeTemplate("TensorTemporary(inputs[0]).value().${name}(${args})")
+CALL_METHOD = CodeTemplate("(${first}).${name}(${args})")
 
 CONSTRUCTOR = CodeTemplate("""\
 {"${descriptor}", [](Node *node) {
   ${assignments}
-  return TensorOp([=](const list_of_retainable & inputs,
-                      list_of_retainable & outputs) {
+  return TensorOp([=](Stack & stack) {
     autograd::profiler::RecordFunction record("${name}");
-    AutoGPU device_guard(deviceForInputs(inputs));
-    pack_list(outputs, ${call});
+    AutoGPU device_guard(deviceForInputs(stack, ${num_inputs}));
+    auto result = ${call};
+    drop(stack, ${num_inputs});
+    pack(stack, std::move(result));
```

Comment:
This is purely because Stack is really just `std::vector<at::Tensor>` and it was easier to supplement it with stack functions than create an entire wrapper around the vector.

Reply:
yea I realized this only later. nvm the comment